ogr/services/github/auth_providers/token.py

```diff
@@ -9,9 +9,9 @@
 
 
 class TokenAuthentication(GithubAuthentication):
-    def __init__(self, token: str, **_) -> None:
+    def __init__(self, token: str, max_retries=0, **_) -> None:
         self._token = token
-        self._pygithub_instance = github.Github(login_or_token=token)
+        self._pygithub_instance = github.Github(login_or_token=token, retry=max_retries)
```

Comment:
What about being able to set the `Retry` object as well as it is possible in case of Pagure:

https://github.com/packit/ogr/blob/135dad28cc85f2f91b8e3b133e3db2c2210d3834/ogr/services/pagure/service.py#L50

Reply:
Actually, both here and in the service class.

Reply:
You're right, it could be handy. Though it must be used wisely.